app/src/main/java/org/sopt/sample/presentation/signup/SignUpActivity.kt

```diff
@@ -0,0 +1,50 @@
+package org.sopt.sample.presentation.signup
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import org.sopt.sample.R
+import org.sopt.sample.data.User
+import org.sopt.sample.databinding.ActivitySignUpBinding
+import org.sopt.sample.presentation.signin.SignInActivity
+import org.sopt.sample.util.showSnackBar
+
+class SignUpActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySignUpBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySignUpBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        onClickComplete()
+    }
+
+    private fun isComplete() {
+        if (isCheck()) {
+            setUser()
+            showSnackBar(binding.root, getString(R.string.signup_complete))
+        } else
+            showSnackBar(binding.root, getString(R.string.signup_failure))
```

Comment:
```suggestion
        showSnackBar(binding.root, getString(if (isCheck()) {
            setUser()
            R.string.signup_complete
        } else {
            R.string.signup_failure
        }))
```

이렇게도 작성해 볼 수 있긴한데 개인적으로는 작성하신 코드가 더 가독성이 좋아보이네용 :)